app/models/user.rb

```diff
@@ -9,4 +9,35 @@ class User < ApplicationRecord
   def info
     {name: name, address: address, city: city, state: state, zip: zip, email: email}
   end
+
+  def all_orders
```

Comment:
This was an easier way to do things, but on my next go-round I think I'll do it with AR.  I don't want to go back and undo all this tho.

Reply:
Sounds like a good plan.